### PR TITLE
Image refresh fixes [no-test]

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: bots/image-refresh [hunk 1]
```diff
@@ -138,6 +139,10 @@ def run(image, verbose=False, **kwargs):
         # we want to validate all image refreshes with external projects
         task.label(pull, ["test-external"])
 
+        # HACK: sometimes statuses get eaten if posting statuses right after
+        # creating the pull request
+        time.sleep(5)
```
Comment (martinpitt) [inline bots/image-refresh]: Could this become a retry loop rather? I. e. trying to api.get the status afterwards?

File: bots/image-refresh [hunk 2]
```diff
@@ -135,15 +135,15 @@ def run(image, verbose=False, **kwargs):
...
+
         # Trigger this pull request
         head = pull["head"]["sha"]
         api = github.GitHub()
         for trigger in triggers:
             api.post("statuses/{0}".format(head), { "state": "pending", "context": trigger,
                 "description": github.NOT_TESTED_DIRECT })
-        # pre-creating statuses circumvents tests-scan's detection of whether bots/ changed
-        # we want to validate all image refreshes with external projects
-        task.label(pull, ["test-external"])
```
Comment (martinpitt) [inline bots/image-refresh]: Please add the "why" to the commit message, not just the "what".

File: bots/image-refresh (hunk 1 above)
Comment (Gundersanne) [reply bots/image-refresh]: of course

File: bots/image-refresh (hunk 2 above)
Comment (Gundersanne) [reply bots/image-refresh]: ack

**Pull request merged**